### PR TITLE
Gives the microwave sprite a little tune-up

### PR DESCRIPTION
**Pull request by Wallemations**
## About The Pull Request
Updates the microwave sprite to be a little more polished 

https://user-images.githubusercontent.com/66052067/192667452-186db160-f62a-469c-a8fa-d47112b3f974.mp4

image
image

## Why It's Good For The Game
image
Look at that icky outline! And that lack of depth! Blegh! Get it outta my sight!!!

## Changelog

:cl: Wallem
imageadd: Gives the microwave a fresh coat of paint
/:cl:

### CODE REVIEW

**Comment by Time-Green** [comment]
I think the old color palette looked a bit better, right now it just blends with the table

**Comment by JohnFulpWillard** [comment]
microwaves are expensive i dont need such a high end one wtf

**Comment by Zonespace27** [comment]
New sprite's better but the old color palette feels better than table-white

**Comment by SuperSayu** [comment]
Maybe add colored microwaves?  I want mine hotrod-red

File: code/modules/food_and_drinks/machinery/microwave.dm [hunk 1]
```diff
@@ -246,6 +246,9 @@
 		var/atom/movable/AM = i
 		AM.forceMove(drop_location())
 	ingredients.Cut()
+	flick("[dirty == 100? "mwbloodyo" : "mwo"]", src)
```
Comment (Mothblocks) [inline code/modules/food_and_drinks/machinery/microwave.dm:249]:
```suggestion
	flick("[dirty == 100 ? "mwbloodyo" : "mwo"]", src)
```

File: code/modules/food_and_drinks/machinery/microwave.dm [hunk 2]
```diff
@@ -372,6 +375,8 @@
 	set_light(0)
 	soundloop.stop()
 	update_appearance()
+	flick("[dirty == 100? "mwbloodyo" : "mwo"]", src)
```
Comment (Mothblocks) [inline code/modules/food_and_drinks/machinery/microwave.dm:378]:
```suggestion
	flick("[dirty == 100 ? "mwbloodyo" : "mwo"]", src)
```

**Comment by Wallemations** [comment]
image
Oh boy I get to redo the blood and destroyed sprites haha yaaaay

Seriously though the aseprite resize tool actually did something good for once
image

**Comment by Wallemations** [comment]
raised the size again since twaticus pointed out the smaller one ruined the effect of the door mesh
I've got some ideas for that small one though so I'm not gonna totally scrap it, but that's for an entirely different PR in the future

**Comment by MrDoomBringer** [comment]
The small one looks way better, if the mesh is messy just widen the sprite 1px

**Comment by Wallemations** [comment]
> The small one looks way better, if the mesh is messy just widen the sprite 1px
> 
> 

That's another hour of editing sprites so I'm gonna get more opinions instead of jumping straight into this one this time

**Comment by Mothblocks** [comment]
@tgstation/sprite-team can u guys please give wall some direction on the microwave size they are goin crazy

**Comment by Mothblocks** [comment]
FWIW I prefer larger microwave because it means I can make my change that shows the food spinning inside the microwave DO NOT STEAL

**Comment by Wallemations** [comment]
> @Wallemations 
> 
> image
> 
> 

Aight then I'll leave it as-is so you can do your thing

**Comment by Mothblocks** [comment]
Well still need a pass from the art team, that just means they are good with it being big

**Review by Mothblocks — APPROVED** [review]

**Comment by Wallemations** [comment]
> Well still need a pass from the art team, that just means they are good with it being big

Ye I just meant _**I'm**_ alright with it being merged in its current state

**Pull request merged**